snapcraft/_store.py

```diff
@@ -281,3 +282,82 @@ def download(snap_name, channel, download_path, arch):
         raise RuntimeError(
             'Failed to download {} at {} (mismatched SHA)'.format(
                 snap_name, download_path))
+
+
+def _get_snap_data(name, store):
+    search_results = store.cpi.search_package(name, 'stable', None)
+    if search_results is None:
```

Comment:
is there a scenario where another zero value would make `if not search_results` not a desirable way to write this?